.drone.yml

```diff
@@ -2,21 +2,40 @@ kind: pipeline
 type: docker
 name: default
 
+environment:
+  CUDA_VERSION: "10.1"
+  PYTORCH_VERSION: "1.6"
+  CUDNN_VERSION: "7"
+  TORCH_CUDA_ARCH_LIST: "5.2 6.0 6.1 7.0 7.5+PTX"
```

Comment:
Isn't this enough ? https://github.com/NVIDIAGameWorks/kaolin/blob/master/tools/linux/Dockerfile#L46